### PR TITLE
Create Skills and Bot: Minimized space needed for dropdown choices and improved (i) icon

### LINKED ISSUE #2741: Create Skills and Bot: Minimize space needed for dropdown choices and improve (i) icon
Category, language and skills name should require less space. In an earlier version of the page the form field names were listed above the field.

1. Please implement it in the same way again.
Something similar to this:
Screenshot from 2019-07-31 01-47-29

2. Regarding the i - icon
* Increase the size
* Make it blue
* For the tool tip and link displayed use the same colors as used on the top menu dropdown

### PR DESCRIPTION
**Pull request by samagragupta**
Fixes #2741 

Changes: Create Skills and Bot: Minimized space needed for dropdown choices and improved (i) icon

Demo Link: https://pr-2746-fossasia-susi-web-chat.surge.sh

Screenshots of the change: [Add screenshots depicting the changes.]

### CODE REVIEW

**Comment by mariobehling** [comment]
1. Please check how the commit message box works on the same page. Could you change it to work in the same way, please?
2. Shorten the width of the text form field for languages a bit.
3. Increase the size a) of the (i) icon even a bit more and b) the text font that comes up when hovering over (i).
4. Also link the (i) itself to https://github.com/fossasia/susi.ai/blob/master/docs/Skill_Tutorial.md

**Comment by mariobehling** [comment]
@fragm3 @samagragupta Please coordinate each others work in future. I see there are overlapping PRs. @fragm3 Let's just integrate @samagragupta 's work. @samagragupta Could you quickly make the changes, please?

**Comment by harsh-jindal** [comment]
@samagragupta Fix the skill name in ipad view.
Screenshot 2019-07-31 at 9 04 02 PM

**Comment by samagragupta** [comment]
@mariobehling @harsh-jindal Done the changes.

File: src/components/cms/SkillCreator/Animation.min.css [hunk 1]
```diff
@@ -1 +1,5 @@
 .iconbutton,.iconbutton>div,.iconbutton>div>div,.iconbutton>svg{transition:none!important}
+.__react_component_tooltip.type-dark {
```
Comment (akshatnitd) [inline src/components/cms/SkillCreator/Animation.min.css]: Why do you do it again and again? When we are moving away from CSS, why use it?

When we already have Tooltip component in Material-UI, then why go for fancy and poor implementations.

**Comment by samagragupta** [comment]
Creating a new PR due to many conflicts.

**Pull request closed without merging**